styles/_header.scss

```diff
@@ -1,42 +1,43 @@
 /***** Header *****/
-
-
 .header {
   background-color: $white;
   margin: 0 auto;
   max-width: $max-width-page-container;
+  box-shadow: 0 3px 5px hsla(0,0%,72.2%,.15);
 
   &__inner {
-    @include max-width-container;
     position: relative;
     align-items: center;
     display: flex;
-    @include fluid-value( height, $min-width, $max-width, 100px, 138px);
+    @include fluid-value( height, $min-width, $max-width, $header__height--min, $header__height--max);
     justify-content: space-between;
   }
 }
 
 .logo {
 
-  &__image {
-    @include fluid-value( max-height, $min-width, $max-width, 25px, 35px);
-    @include tablet { 
-      padding-right: 16px;
-      border-right: solid 2px $primary;
-      margin-right: 16px;
+  &__link {
+    &:hover {
+      text-decoration: none;
     }
+  }
+
+  &__image {
+    @include fluid-value( max-height, $min-width, $max-width, 18.50px, 35px);
```

Comment:
`18.50px` should be rounded. There is no such thing as a real half pixel

Reply:
_facepalm_